src/UI/Buyer/src/app/profile/containers/payment-list/payment-list.component.ts

```diff
@@ -45,7 +58,7 @@ export class PaymentListComponent implements OnInit, OnDestroy {
     this.authorizeNetSerivce.CreateCreditCard(card).subscribe(
```

Comment:
I think its a bit too opinionated to have a specific credit card integration in the base app, I had meant to remove authNet completely. We should leave it up to the developer to decide which integration to use.

Reply:
I do think its important to showcase how to use credit cards on OC. It was a harder topic for me to learn. How can we create credit card functionality without some integration?

Reply:
To be clear I'm not saying we shouldn't allow usage of credit cards at all, we'll just exclude the non-ordercloud parts of using a credit card (authorization etc). I think this is one of the components that will benefit from being well commented. 

You can still show how list credit cards to select, create a payment with Type CreditCard and the CreditCardID, the only thing that would be mocked is that payment.Accepted = true will have to be done in the application and requires an elevated role. We should mention that this is something that is typically done by a payment processor once the credit card has been authorized.

Reply:
you can check out angular-buyer (angularjs) to see kind of how it was built there